CHANGES.txt

```diff
@@ -1,4 +1,4 @@
-v0.3.2, 2014-05-01 Update configparser lib version to 3.5.0b2
```

Comment:
one more small fix please, you actually erased the changelog for the previous version 0.3.2, what I meant here was to:

1. add a new line on top describing the changes in your fix, something like: `v0.3.3, 2017-09-05 fixed html parsing, restored functionality`
2. preserve the previous line on 0.3.2